native/src/modules/layout/components/Header.js

```diff
@@ -142,8 +131,8 @@ class Header extends React.PureComponent<PropsType> {
   }
 
   render () {
-    const { routeCityModel, navigation, t, theme, goToLanguageChange, peeking, categoriesAvailable } = this.props
-    const sharePath = navigation.getParam('sharePath')
+    const { routeCityModel, scene, t, theme, goToLanguageChange, peeking, categoriesAvailable } = this.props
+    const sharePath = scene.route.params?.sharePath || ''
```

Comment:
I think it makes more sense to pass null if the share path is undefined instead of an empty string.

Reply:
Will fix in #105